test/angular1-test.js

```diff
@@ -0,0 +1,69 @@
+import { Angular1Selector } from '../lib';
```

Comment:
What do you think about an idea to write in the **Usage** section of `Readme` something like:
```md
See more examples [here](<a link to this file with tests>)
```

/cc @VasilyStrelyaev @miherlosev